### DISCUSSION TITLE
github  gnash <title>

### DISCUSSION THREAD

**Discussion by Miperro3** [community/community#46203] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

After coding the github gnash into terminal on linux mint 21.1  it says no such file!!!  Gnash is needed to view swf files!  Please advise how to do it.  tnx very much

**Comment by jge162** [comment]
Hello @Miperro3 

first run this command to verify that gnash is installed ->
`dpkg -l | grep gnash`

only if it shows it is not installed do these commands next ->

`sudo apt-get update`
`sudo apt-get install gnash`

if it shows that it is installed check path ->
`whereis gnash`

Hope this helps. I think Gnash is not installed.

**Comment by Miperro3** [comment]
Hello Jermyiah
Here is what showed:     But look at how it shows it in my home folder.




On Thursday 02 February 2023 03:36:04 PM (-08:00), Jermyiah wrote:



Hello @Miperro3

first run this command to verify that gnash is installed ->
dpkg -l | grep gnash

only if it shows it is not installed do these commands next ->

sudo apt-get update
sudo apt-get install gnash

if it shows that it is installed check path ->
whereis gnash

Hope this helps. I think Gnash is not installed.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.


-- 
Sent with Vivaldi Mail. Download Vivaldi for free at vivaldi.com

**Reply by jge162** [reply]
if you replied directly via email, I cannot see anything sorry. You must log into GitHub and upload pictures to chat directly.

**Reply by Miperro3** [reply]
Here are the 2 files:

![gnash](https://user-images.githubusercontent.com/122307794/216484374-9daeab1b-4fc5-4a1b-bada-823fa5c51
![gnash msg](https://user-images.githubusercontent.com/122307794/216484392-785bc356-76de-4e40-a844-b4916129de24.png)
968.png)

**Reply by jge162** [reply]
did you run update first? seems like gnash is no longer supported maybe try this search -> apt-cache search gnash

**Reply by Miperro3** [reply]
gnash is no longer suported since a couple of years ago.  but i expect you computer geniuses to compile something to make it work.  ok?

**Reply by jge162** [reply]
If it is no longer supported -> why don't you try Lightspark instead
`sudo apt-get install lightspark`

**Reply by Miperro3** [reply]
This is what happened:
![lightspark](https://user-images.githubusercontent.com/122307794/216486528-c13ca45f-b467-4d63-8f29-7ad2565dd65a.png)

**Reply by jge162** [reply]
Many web browsers, such as Google Chrome, Mozilla Firefox, and Microsoft Edge, have built-in support for viewing SWF files without the need for a plugin.

**Reply by Miperro3** [reply]
Hello jge162
Success!  First I tried ruffle which worked, but poorly and not worth using for me.  Then I used firefox and it asked what program to use to open the swf file.  It gave me the option of the gnash that I had downloaded but 
 had not worked because I had not put in the full name after gnash ie: gnash-raymii.  Then I simply used that same option to view my swf files without using a browser and it is working perfectly for me at this time.  Many tnx for the help.  All systems go at this time!!!

**Reply by jge162** [reply]
Sweet. Good job.

**Comment by Miperro3** [comment]
I don[t know how to chat directly on github.


On Thursday 02 February 2023 04:45:19 PM (-08:00), Jermyiah wrote:



if you replied directly via email, I cannot see anything sorry. You must log into GitHub and upload pictures to chat directly.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.


-- 
Sent with Vivaldi Mail. Download Vivaldi for free at vivaldi.com

**Reply by jge162** [reply]
you did it to create the post. you went to Github's website. or click at the bottom of the email that says view on GitHub website.
